new-log-viewer/src/components/DropFileContainer/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {
+    useContext,
+    useState,
+} from "react";
+
+import {StateContext} from "../../contexts/StateContextProvider";
+import {CURSOR_CODE} from "../../typings/worker";
+
+import "./index.css";
+
+
+interface DropFileContextProviderProps {
+    children: React.ReactNode;
+}
+
+/**
+ * A container element to add drag & drop functionality to the child elements.
+ *
+ * @param props
+ * @param props.children
+ * @return
+ */
+const DropFileContainer = ({children}: DropFileContextProviderProps) => {
+    const {loadFile} = useContext(StateContext);
+    const [isFileHovering, setIsFileHovering] = useState(false);
+
+    /**
+     * Handler for a drag event.
+     *
```

Comment:
Probably don't need the description, but I don't mind if you want to keep it.

Reply:
Removed for now since the code simply wraps around `setIsFileHovering()`.